packages/url_launcher/url_launcher_web/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # 0.1.1+2
 
-- Open urls with "_top" (instead of "_blank") as target on iOS PWAs.
+- Open urls with target "_top" (instead of default "_blank") on iOS PWAs.
```

Comment:
But the default is `''`, right? :P

Reply:
I might be wrong but most browsers treat "no valid" names (empty string is an instance of this) as "_blank" by default. TBH I could not find a valid reference that enunciated this as standard so I think its safer to only indicate the change we are introducing.